integration-tests/search.test.ts

```diff
@@ -56,6 +67,55 @@ describe('search', () => {
         }
     });
 
+    test('date ranges', async () => {
+        const randomPatientData = randomPatient();
+        randomPatientData.birthDate = '1990-05-05';
+        const testPatient: ReturnType<typeof randomPatient> = (await client.post('Patient', randomPatientData)).data;
+
+        // wait for the patient to be asynchronously written to ES
+        await waitForPatientToBeSearchable(client, testPatient);
+
+        const aFewMinutesAgo = new Date(Date.now() - 1000 * 60 * 10).toJSON();
+        const p = (params: any) => ({ url: 'Patient', params: { _lastUpdated: `ge${aFewMinutesAgo}`, ...params } });
+
+        const testsParamsThatMatch = [
+            p({ birthdate: '1990-05-05' }),
+            p({ birthdate: 'eq1990-05-05' }),
+            p({ birthdate: 'eq1990' }),
+            p({ birthdate: 'gt1990-05-01' }),
+            p({ birthdate: 'ge1990-05-04' }),
```

Comment:
Could we test the 'equal to' part?
```suggestion
            p({ birthdate: 'ge1990-05-05' }),
```